pulp_ansible/app/serializers.py

```diff
@@ -95,6 +95,21 @@ class CollectionRemoteSerializer(RemoteSerializer):
         required=False,
         allow_null=True,
     )
+    auth_url = serializers.CharField(
+        help_text=_(
+            "The URL of a Keycloak server ‘token_endpoint’ "
```

Comment:
```suggestion
            "The URL to receive a session token from, e.g. used with Automation Hub."
```

Can we also add a sentence like: `see https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client for more details.`